src/Components/Artwork/SaveButton/useSaveArtworkToLists.ts

```diff
@@ -115,21 +114,41 @@ export const useSaveArtworkToLists = (options: SaveArtworkToListsOptions) => {
       return ResultAction.ShowingAuthDialog
     }
 
-    if (savedListId || artwork.isSavedToCustomLists) {
-      openSelectListsForArtworkModal()
-      return ResultAction.SelectingListsForArtwork
-    }
-
-    const response = await saveToDefaultList()
-    const isSaved = response?.saveArtwork?.artwork?.isSaved
+    if (!artwork.isSavedToAnyList) {
```

Comment:
This is the 'meat' of the PR where the 'unsave' flow has shifted to first fetch custom list inclusion status to know whether or not to pop up the modal.